docs/io.cozy.files_metadata.md

```diff
@@ -145,7 +144,8 @@ expected in every cases.
 - `bankName` : {string} name of the related bank.
 - `carbonCopy`: {boolean} : if the document is the original document imported by the connector
 - `electronicSafe`: {boolean} : if the document is secured in a secure storage
-
+- `backupDeviceIds`: {string[]} : id of the devices owning the backup
```

Comment:
I wonder if we should add the precision that it's only for folder, like: 
``backupDeviceIds`: {string[]} id of the devices owning the backup. Only for directories.`

Reply:
Done. It is still unsure if we will use it for files too. But for the moment I use it only for folders so it is better if explicit.